SlidingSegmentedControlTests/SlidingSegmentedControlTests.swift

```diff
@@ -123,6 +125,18 @@ class SlidingSegmentedControlTests: XCTestCase {
         XCTAssertEqual(slidingSegmentedControl.selectionView.layer.cornerRadius, expectedCornerRadius)
     }
 
+    func testSelectionViewDefaultFrame() {
+        let expectedFrame = slidingSegmentedControl.buttons[0].frame
+        XCTAssertEqual(slidingSegmentedControl.selectionView.frame, expectedFrame)
+    }
+
+    func testSlectionViewFrame() {
```

Comment:
@salutis I'm having some troubles with testing frames of selection view. This view should slide between buttons when button is being tapped/selected and should expand on sliding gesture. Here I want to test that after setting `selectedSegment` frame of `selectionView` is equal to button's frame. But since we're initializing with zero frame, our view has zero frame, and it's subviews has zero frame, so all buttons have zero frame and all tests pass. Any ideas here?